src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+$card-width: 200px;
+$row-gap: 46px;
```

Comment:
Make sure that you use these variables instead of hardcoded values. Also, would be great to add more variables